doc/github_actions/security_issues.rst

```diff
@@ -0,0 +1,102 @@
+security-issues
+===============
+
+Example Usage
+-------------
+
+.. code-block:: yaml
+
+    name: Report Security Issues for Repository
+
+    on:
+      schedule:
+        # “Every day at 00:00.” (https://crontab.guru)
+        - cron: "0 0 * * *"
+
+    jobs:
+
+      report_security_issues:
+
+        name: Report Security Issues
+
+        runs-on: ubuntu-latest
+
+        permissions:
+          issues: write
+
+        steps:
+          - name: SCM Checkout
+            uses: actions/checkout@v2
+
+          - name: Report Security Issues
+            uses: exasol/python-toolbox/.github/actions/security-issues@0.6.0/security-issues-action
+            with:
+              format: "maven"
+              command: "cat maven-cve-report.json"
+              github-token: ${{ secrets.GITHUB_TOKEN }}
+
+Configuration
+-------------
+This action exposes 3 configuration parameters `command`_, `format`_ and `github-token`_, for details see
+the specific sections below.
+
+command
++++++++
+
+Workspace command which shall be executed in order to check the project's dependencies for CVEs.
+
+.. note::
+
+    The calling workflow need to make sure the specified command can be executed in the context of the workflow.
+
+
+format
+++++++
+
+Specifies converter which needs to be applied on the output of the provided command.
+Currently there are only two converters available
+
+#. maven
+
+    Converts the output of mavens oss plugin into required input format.
+
+
+#. pass-through
+
+    In case the command itself already outputs the expected input format, the format can be specified as code:`pass-through`.
+
+
+Input Format
+------------
+
+The expect intput format is jsonl (line based json), of the following form:
+
+.. code-block:: python
+
+    { "cve": "<cve-id>", "cwe": "<cwe-id>", "description": "<multiline string>", "coordinates": "<string>", "references": ["<url>", "<url>", ...] }
+
+
+.. attention::
+
+    The input format may change in the future. Therefore make sure to rather use or contribute a converter for
+    a specific format rather than outputting this format by your own tooling.
+
+
+github-token
+++++++++++++
+The temporary github token of the workflow needs to be passed into the action (:code:`${{ secrets.GITHUB_TOKEN }}`),
+in order to enable the action to query and created github issue.
+
+
+Todo's
```

Comment:
Create tickets for todo

Reply:
Not so sure about this, this are "potential" todo's. Time will tell if they make sense.
So imho it's good to have this written down, but convoluting the issue tracker with 
them yet does not make sense imho.

Reply:
Rename to Ideas